torch/_torch_docs.py

```diff
@@ -8900,18 +8900,23 @@ def merge_dicts(*dicts):
 
 add_docstr(torch.argsort,
            r"""
-argsort(input, dim=-1, descending=False) -> LongTensor
+argsort(input, *, dim=-1, descending=False, stable=True) -> Tensor
```

Comment:
I think when stable is not specified it should be not a stable routine, no?

Reply:
Yup, changed; that's a great point, but also something which the [array_api disagrees on](https://data-apis.org/array-api/latest/API_specification/generated/signatures.sorting_functions.argsort.html), suggests that the default is meant to be `True`.